index.css

```diff
@@ -37,4 +37,9 @@ button {
   width: 50%;
   cursor: pointer;
   margin: auto;
+}
+
+.gitButton{
+	 background-color: green;
+  color: white;
```

Comment:
Please Fix the indentation on this line.

Reply:
Thanks